plugin/trino-cassandra/src/test/resources/cu-cassandra-latest.yaml

```diff
@@ -0,0 +1,23 @@
+cluster_name: 'Test Cluster'
+partitioner: org.apache.cassandra.dht.Murmur3Partitioner
+data_file_directories:
+    - /var/lib/cassandra/data
+commitlog_directory: /var/lib/cassandra/commitlog
+saved_caches_directory: /var/lib/cassandra/saved_caches
+commitlog_sync: periodic
+commitlog_sync_period: 10000ms
+commitlog_segment_size: 32MiB
+seed_provider:
+    - class_name: org.apache.cassandra.locator.SimpleSeedProvider
+      parameters:
+          - seeds: "127.0.0.1:7000"
+concurrent_reads: 2
+concurrent_writes: 2
+concurrent_counter_writes: 2
+concurrent_materialized_view_writes: 2
+commitlog_total_space: 8192MiB
```

Comment:
8 GB sounds like a lot.
where is it stored?

Reply:
`commitlog_total_space: Total space to use for commit logs on disk.`
`Default Value: 8192MiB`
https://cassandra.apache.org/doc/latest/cassandra/architecture/storage_engine.html

For the test, I think we can reduce it to 1 MB same similar to https://github.com/trinodb/trino/blob/master/plugin/trino-cassandra/src/test/resources/cu-cassandra.yaml#L226.